docs/contributing/Building, Debugging, and Testing on Windows.md

```diff
@@ -167,6 +167,15 @@ You can also attach a debugger to Visual Studio and check the loaded modules, lo
 where the various `CodeAnalysis` modules were loaded from (the `RoslynDev` should load them somewhere 
 under `AppData`, not from `Program File`).
 
+### Testing on the `runtime` repo
+
+1. make sure that you can build the `runtime` repo as baseline (install some pre-requisites then `build.cmd`)
```

Comment:
```suggestion
1. make sure that you can build the `runtime` repo as baseline (run `build.cmd libs`, installing any prerequisites if prompted to)
  - see https://github.com/dotnet/runtime/blob/master/docs/workflow/building/libraries/README.md for more details
```

feeling less confident in linking directly into runtime repo docs, since we won't quickly discover if the links break, but it will probably help overall.